### DISCUSSION TITLE
Error boot file not found

### DISCUSSION THREAD

**Discussion by 216321** [antonym/netboot.xyz#1371] [General]
**Describe the bug**
I have the docker image installed on Gen 1 HyperV VM of Ubuntu Server 22.04with a bridged adapter and a static MAC lining up with a DHCP reservation that the DHCP server points to for PXE boot and set for the. The issue is that it isn't finding the boot file.
**To Reproduce**
1. Install netbootxyz docker according to the netboot.xyz site
2. pxeboot a device on the same network

**Expected behavior**

Get error of file not found.
**Screenshots**

![image](https://github.com/netbootxyz/netboot.xyz/assets/110435555/9272bf2e-c038-4605-be5c-97fbf30002d7)

**Additional context**

Both machines are being picked up by the network and it has bin verified the net boot server is picking up the DHCP reservation.

**Comment by antonym** [comment]
Make sure your TFP ports are open and accessible to the outside and that you have a next-server set.

**Comment by anthonyuk** [comment]
I’m self hosting and followed the guide for this but in OPNSense DHCP NetBoot settings I have to set the boot file format as ipxe/bootfile.name for it to work.

After this change it is fine for me so not sure if this is how it is supposed to work or if there is something wrong on my side.
The TFTP /Netboot host (Proxmox CT) is not accessible from a browser but serves TFTP fine also.
